poucave/html/style.css

```diff
@@ -29,10 +29,15 @@ body {
   color: var(--text-color);
 }
 
-a, a:hover, a:visited {
+a[href^="+"], a:hover[href^="+"], a:visited[href^="+"] {
   color: var(--link-color);
 }
 
+#svg-diagram {
```

Comment:
I don't think we should enforce a `max-height`.   By default if the SVG 404's ... will the height be 0px?

Reply:
Yes height will be 0px.

I thought the max-height would make sense with `width:100%`. It basically tries to fit horizontally without taking too much vertical space. I changed to 33% of vertical screen instead of static 400px